charts/openfga/templates/deployment.yaml

```diff
@@ -406,25 +406,52 @@ spec:
           readinessProbe: {{- include "common.tplvalues.render" (dict "value" .Values.customReadinessProbe "context" $) | nindent 12 }}
           {{- else if .Values.readinessProbe.enabled }}
           readinessProbe: {{- include "common.tplvalues.render" (dict "value" (omit .Values.readinessProbe "enabled") "context" $) | nindent 12 }}
+          {{- if .Values.grpc.tls.enabled }}
             exec:
-              command: ["grpc_health_probe", "-addr={{ .Values.grpc.addr }}"]
+              command:
+              - grpc_health_probe
+              - -addr={{ .Values.grpc.addr }}
+              - -tls
+              - -tls-no-verify
```

Comment:
I would add this one behind a new value, maybe something like  `.Values.grpc.tls.health_probe.tls_no_verify`. 

Not verifying the certificate is not really the intended behaviour I believe, since this is set to "false" by default in `grpc_health_probe`.

Reply:
(to address the other issue, maybe we need to include a way to pass `-tls-server-name` as well)